website/docs/components/app-footer/partials/code/how-to-use.md

```diff
@@ -0,0 +1,64 @@
+## How to use this component
+
+The `AppFooter` is used to contain a brief list of useful links and information relevant to an entire application. 
+
+### Basic invocation with recommended content
+
+The `AppFooter` contains only a copyright by default. The provided `LegalLinks` child component is recommended for inclusion on HashiCorp Cloud application footers.
```

Comment:
We discussed previously that they are mostly relevant to Cloud applications so we may still want to mention that. @didoo @Dhaulagiri what do you think?

Reply:
A slight adjustment like this might be a bit more clear:

> recommended for inclusion in application footers for our cloud products like HCP and TFC.

Reply:
Thanks!

Reply:
@jorytindall Does that sound good? Any more massaging of the wording needed do you think? If you want to update your suggested change then I can apply it.

Reply:
Nope, that I think that makes the statement a bit more broad while focusing on the specific use cases 👍

Reply:
I applied the version recommended by Brian. Going to resolve this thread, thanks